### PR TITLE
bump sklearnserver rock to 0.11.2

### LINKED ISSUE #35: Create a rock for `kserve/sklearnserver:v0.11.2`
### Context

Create a rock for KServe's sklearnserver based on its upstream [Dockerfile](https://github.com/kserve/kserve/blob/v0.11.2/python/sklearn.Dockerfile).

### What needs to get done

1. Create the rockcraft project that will be equivalent to the dockerfile

### Definition of Done

The rockcraft project is merged in `main` and the resulting container image is published in the `charmedkubeflow` container image registry.

### PR DESCRIPTION
**Pull request by ca-scribner**
closes #35 

### Tester instructions

Launch the server with:
```
# download the model locally
gsutil cp -r gs://kfserving-examples/models/sklearn/1.0/model ./sample_model/

# mount the model into the container at runtime
docker run -p 8080:8080 -v $(pwd)/sample_model:/mnt/models sklearnserver:0.11.2 --model_name test_model --model_dir=/mnt/models --http_port=8080

```

Test the server with:
```
cat <<EOF >> iris-input-v2.json
{
  "inputs": [
    {
      "name": "input-0",
      "shape": [2, 4],
      "datatype": "FP32",
      "data": [
        [6.8, 2.8, 4.8, 1.4],
        [6.0, 3.4, 4.5, 1.6]
      ]
    }
  ]
}
EOF

curl -v \
  -H "Content-Type: application/json" \
  -d @./iris-input-v2.json \
  localhost:8080/v2/models/test_model/infer
```

### CODE REVIEW

File: sklearnserver/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+## Testing instructions
+
+From the [upstream usage example](https://kserve.github.io/website/master/modelserving/v1beta1/sklearn/v2/#deploy-the-model-with-rest-endpoint-through-inferenceservice/), this rock can be tested locally using:
+
+Launch the server with:
+```
+# download the model locally
+mkdir sample_model
+gsutil cp -r gs://kfserving-examples/models/sklearn/1.0/model ./sample_model/
```
Comment (DnPlas) [inline sklearnserver/README.md:9]: Perhaps we need a small pre-requisites section with this, docker, and any other sw that is used for testing OR provide a `requirements.txt` file that people can just use. Up to you, either works for me.

**Review by DnPlas — COMMENTED** [review]
Thanks @ca-scribner, super tiny little comment, other than that LGTM.

File: sklearnserver/README.md (hunk 1 above)
Comment (ca-scribner) [reply sklearnserver/README.md:16]: Added a prereqs section - wdyt?

**Review by DnPlas — COMMENTED** [review]
Same comment as [here](https://github.com/canonical/kserve-rocks/pull/41/files#r1510728987).

I think that this PR could be split in two just to unblock #35 and then we can move the discussion about the testing instructions elsewhere. wdyt?

**Review by DnPlas — APPROVED** [review]

**Pull request merged**